src/main/java/edu/byu/cs/util/RepoUrlValidator.java

```diff
@@ -31,7 +40,7 @@ public static String clean(@Nullable String repoUrl) throws InvalidRepoUrlExcept
      * @return Cleaned and standardized repository URL.
      * @throws InvalidRepoUrlException If the repo URL is invalid.
      */
-    public String cleanRepoUrl(@Nullable String repoUrl) throws InvalidRepoUrlException {
+    public static String cleanRepoUrl(@Nullable String repoUrl) throws InvalidRepoUrlException {
```

Comment:
Changing this to a static method nullifies the benefits and effects of object-oriented programming. Whereas it was previously extensible and adjustable, making the method static removes that ability without any other improvements. This should be reverted.